libs/liblobby/lobby/interface.lua

```diff
@@ -648,7 +648,128 @@ Interface.commands["FRIENDREQUESTLISTEND"] = Interface._OnFriendRequestListEnd
 -- Battle commands
 ------------------------
 
+
+-- if num1 > num2 return 1
+-- if num1 < num2 return 0
+-- if num1 = num2 return -1
```

Comment:
In most programming languages where there is function like this (Java, C#) it's called compare and the logic is
x < y -> -1
x == y -> 0
x > y -> 1

so it's kind of monotonic